internal/command/jsonformat/differ/replace/replace.go

```diff
@@ -0,0 +1,63 @@
+package replace
+
+import "encoding/json"
+
+type ForcesReplacement struct {
```

Comment:
It'd be nice to have some doc comments about the semantics behind this type, its relationship to the JSON data `Parse` supports, and how it's used.

Reply:
Done!